src/config/configuration.ts

```diff
@@ -161,9 +166,9 @@ const configuration = () => {
  * Registers built-in JobActions. Should be called exactly once.
  */
 export function registerDefaultActions() {
-  registerCreateAction(LogJobAction.actionType, (data) => new LogJobAction(data));
-  registerUpdateAction(LogJobAction.actionType, (data) => new LogJobAction(data));
-  registerCreateAction(EmailJobAction.actionType, (data) => new EmailJobAction(data));
+  registerCreateAction(LogJobAction);
+  registerUpdateAction(LogJobAction);
+  registerCreateAction(EmailJobAction);
```

Comment:
@sbliven I am still confused why we want to register the actions? and if this is the best practice in typescript.